doc/sharing.rst

```diff
@@ -0,0 +1,161 @@
+Sharing tools, projects and publications
```

Comment:
Let's not duplicate all this content. Simply reference the existing contributors guide and expand it if necessary.